Sources/Components/Colors.swift

```diff
@@ -0,0 +1,49 @@
+import UIKit
+
+public extension UIColor {
+
+	static let gray800 = UIColor(0x101418)
```

Comment:
The indentation on this file seems wrong, but on Xcode it looks right

Reply:
May have something to do with your Xcode indention settings for this project. Mine are set to prefer indent using spaces, tab and indent width both set to 4.

Reply:
Mine was set to tabs, that was it!